src/DazCli.js

```diff
@@ -11,9 +12,12 @@ class DazCli extends Daz {
     _run(args, input, output, terminal) {
         if (args.length >= 3) {
             const command = args[2]
-    
+
             if (command === '--help') {
-                this.output.write(`${fs.readFileSync(path.join(__dirname, 'help.txt'))}\n`)
+                exec(`sh ${path.join(__dirname, 'help.sh')}`,
+                    (err, stdout, stderr) =>
+                    err ? this.output.write(`${fs.readFileSync(path.join(__dirname, 'help.txt'))}\n`) : this.output.write(`${stdout}`)
```

Comment:
THIS code
this.output.write(`${stdout}`)
doesn't seem to have sense...
have you tested it?
WHAT i meant in the message before is that - if you follow index.js - output is already stdout. so "stdout writing stdout" doesn't seem to have sense. before you had a "console.log(stdout)", but now has disappeared and replaced by this new code. maybe you wanted to use "terminal"? (see input params)

Reply:
super ,) 👍